homeassistant/components/modbus/__init__.py

```diff
@@ -265,6 +277,44 @@
                     vol.Optional(CONF_WRITE_REGISTERS, default=False): cv.boolean,
                 }
             ),
+            vol.Optional(CONF_FAN_MODE_REGISTER): vol.Maybe(
+                {
+                    CONF_ADDRESS: cv.positive_int,
+                    CONF_FAN_MODE_VALUES: {
+                        vol.Optional(CONF_FAN_MODE_ON): vol.Any(
+                            cv.positive_int, [cv.positive_int]
```

Comment:
I had only made this a list to ensure consistent configuration options to `CONF_HVAC_MODE_VALUES`.

I agree with you, I can find no real-world examples where this would be needed.